test_conformance/compatibility/test_conformance/basic/CMakeLists.txt

```diff
@@ -39,16 +37,11 @@ set(${MODULE_NAME}_SOURCES
     test_astype.cpp
     test_async_copy.cpp
     test_sizeof.c
-    test_vector_creation.cpp
     test_vec_type_hint.c
-    test_numeric_constants.cpp
     test_constant_source.cpp
     test_bufferreadwriterect.c
     test_async_strided_copy.cpp
-    test_preprocessors.cpp
     test_kernel_memory_alignment.cpp
```

Comment:
This one only has white space diffs. We could remove that too?

Reply:
I will remove all of these in the next change. I'm trying to slice this into small-ish, (kind of) reviewable chunks.